imgs/x.md

```diff
@@ -0,0 +1 @@
+# How yo doin' ??
```

Comment:
What's the point of this file?

Reply:
Sorry, I had to create a folder in the repository. But, I can't create a new folder without any file. So, I had to create a new file named x.md and I didn't know what to type. Therefore, I done this file. Sorry.